src/main/java/com/parsing/parsers/pdf/parsing/PageOCRPreparator.java

```diff
@@ -19,7 +20,7 @@
 import org.springframework.stereotype.Component;
 
 import javax.imageio.ImageIO;
-import java.awt.Rectangle;
+import java.awt.*;
```

Comment:
don't use * please, it causes problems with different imports

Reply:
done

Reply:
done